contnumber.py

```diff
@@ -0,0 +1,29 @@
+import csv
+with open('C://22BCA192//PYTHON//contact1.csv','w',newline='') as obj:
+    p=csv.writer(obj)
+    h=['name','contact']
+    p.writerow(h)

```

Comment:
what is the use of writerow()?

Reply:
The writerow() method is used to write a single row at a time into a CSV file.